unity_cogs/react_report.py

```diff
@@ -0,0 +1,159 @@
+import sys
+import motor
+import logging
+import discord
+import datetime
+
+from discord.ext import commands
+from unity_util import bot_config
+
+
+mongo = motor.motor_asyncio.AsyncIOMotorClient(host=bot_config.MONGODB_HOST, port=int(
+    bot_config.MONGODB_PORT), replicaSet="rs01", username=bot_config.MONGODB_USERNAME, password=bot_config.MONGODB_PASSWORD, authSource=bot_config.MONGODB_DATABASE, authMechanism='SCRAM-SHA-1')
+db = mongo[bot_config.MONGODB_DATABASE]
+
+logging.basicConfig(level=logging.INFO, format="[%(levelname)s]\t %(name)s: %(message)s", handlers=[
+    logging.StreamHandler(sys.stdout),
+    logging.FileHandler(f'./logs/{bot_config.LOGGING_FILENAME}')
+])
+
+
+class ReactReport(commands.Cog):
+    def __init__(self, client):
+        self.client = client
+
+    @commands.Cog.listener()
+    async def on_ready(self):
+        logging.info("Reaction report cog online.")
+
+    @commands.Cog.listener()
+    async def on_raw_reaction_add(self, payload: discord.RawReactionActionEvent):
+        if payload.emoji.name != "⚠️":
```

Comment:
Should the emoji be a config choice? Or at least be defined at the top of the file?